inst/FunctionVisualizer/app.R

```diff
@@ -68,7 +68,7 @@ server <- function(input, output, session) {
     if (!is.null(input$argument)) {
       currentArg <<- input$argument
     }
-    if (v$equation != input$equation) {
+    if (v$equation != input$equation & !is.null(v$snapshots)) {
```

Comment:
The app would crash when changing the equation without having saved any snapshots before.